backend/src/interfaces/Ientities/Icourses.ts

```diff
@@ -1,13 +1,14 @@
-import { TMaterials } from './Imaterials';
+import { ObjectId } from 'mongoose';
 
 interface ICourse {
-  _id?: string;
+  _id?: ObjectId;
   title: string;

```

Comment:
`_id`  also used on frontend interfaces and was `string`. Wont this fix crash out interfaces?

Reply:
it's string